### DISCUSSION TITLE
Unable for my friend to come into my server

### DISCUSSION THREAD

**Discussion by Willy-Whiskers** [Pryaxis/TShock#1870] [Q&A]
I am trying to get my friend into my server via IP but it doesn't seem to work and it just stays on connecting is it broken or am I doing something wrong please tell me what I can do thank you.

**Comment by hakusaro** [comment]
Did you take a look at this one? https://github.com/Pryaxis/TShock/discussions/1717

**Reply by Willy-Whiskers** [reply]
Thanks
